recipe/activate.bat

```diff
@@ -63,19 +63,22 @@ if errorlevel 1 (
     echo Windows SDK version found as: "%WindowsSDKVer%"
 )
 
+
 IF "@{target}" == "win-64" (
-  set "CMAKE_GEN=Visual Studio @{ver} @{year} Win64"
+  set "CMAKE_PLAT=x64"
   set "BITS=64"
 ) else (
-  set "CMAKE_GEN=Visual Studio @{ver} @{year}"
+  set "CMAKE_PLAT=x86"
```

Comment:
The platform name for 32-bit Windows is Win32, see https://cmake.org/cmake/help/latest/generator/Visual%20Studio%2015%202017.html#platform-selection .
```suggestion
  set "CMAKE_PLAT=Win32"
```